portfolio/src/main/webapp/index.html

```diff
@@ -73,53 +73,62 @@ <h1>Makeup Gallery</h1>
        width="650" height="650">
     </div>
   </body>
-</html>
-
-<!-- Slideshow container -->
-<div class="slideshow-container">
+  <!-- Slideshow container -->
+  <div class="slideshow-container">
   <h1>Guinea Pig Gallery</h1>
   <p>Press the arrows to open and view the Guinea Pig Gallery!</p>
   <!-- images with number and caption text -->
-  <div class="slides-fade">
-    <img src="images/GP2.jpg" alt="My guinea pig, Butter, smiling at the camera" width="550" 
-    height="550">
-    <div class="text">My guinea pig, Butter, smiling at the camera</div>
-  </div>
+    <div class="slides-fade">
+       <img src="images/GP2.jpg" alt="My guinea pig, Butter, smiling at the camera" width="550" 
+        height="550">
+        <div class="text">My guinea pig, Butter, smiling at the camera</div>
+    </div>
 
-  <div class="slides-fade">
-    <img src="images/GP1.jpg" alt="My guinea pig, Butter, sniffing the camera" width="550" 
-    height="550">
-    <div class="text">My guinea pig, Butter, sniffing the camera</div>
-  </div>
+    <div class="slides-fade">
+        <img src="images/GP1.jpg" alt="My guinea pig, Butter, sniffing the camera" width="550" 
+        height="550">
+        <div class="text">My guinea pig, Butter, sniffing the camera</div>
+    </div>
 
-  <div class="slides-fade">
-    <img src="images/GP3.jpg" alt="My guinea pig, Koko, posing for the camera" width="550" 
-    height="550">
-    <div class="text">My guinea pig, Koko, posing for the camera</div>
-  </div>
+    <div class="slides-fade">
+       <img src="images/GP3.jpg" alt="My guinea pig, Koko, posing for the camera" width="550" 
+       height="550">
+       <div class="text">My guinea pig, Koko, posing for the camera</div>
+    </div>
 
-  <div class="slides-fade">
-    <img src="images/GP4.jpg" alt="My guinea pig, Taffy, posing for the camera" width="550" 
-    height="550">
-    <div class="text">My guinea pig, Taffy, posing for the camera</div>
-  </div>
+    <div class="slides-fade">
+        <img src="images/GP4.jpg" alt="My guinea pig, Taffy, posing for the camera" width="550" 
+       height="550">
+       <div class="text">My guinea pig, Taffy, posing for the camera</div>
+    </div>
 
-  <div class="slides-fade">
-    <img src="images/GP5.jpg" alt="My guinea pig, Butter, running towards the camera" 
-    width="550" 
-    height="550">
-    <div class="text">My guinea pig, Butter, running towards the camera</div>
-  </div>
+    <div class="slides-fade">
+       <img src="images/GP5.jpg" alt="My guinea pig, Butter, running towards the camera" 
```

Comment:
Nit: Should we only use 2-space indentation?

Reply:
+1, it looks like some of the indents got mixed up during the change.